framework/include/media/DataSource.h

```diff
@@ -91,14 +91,14 @@ class DataSource
 	 * @return The channel count of the stream data.
 	 * @since TizenRT v2.0 PRE
 	 */
-	int getChannels();
+	unsigned short getChannels();
 	/**
```

Comment:
16bit also looks too much for channels,
Would you please consider to use unsigned char?

Reply:
yes, you right.

Reply:
@juitem  fixed it.